app/models/support_day.rb

```diff
@@ -1,4 +1,6 @@
 class SupportDay < ApplicationRecord
+  SUPPORT_TYPES = %w[dev ops ooh1 ooh2].freeze
```

Comment:
```suggestion
  OPSGENIE_SUPPORT_TYPES = %w[dev ops ooh1 ooh2].freeze
```

to be clear about what system these codes are for (or whatever the appropriate label is).